### DISCUSSION TITLE
igxFor  - Disable allowing inner elements to scroll during inertia wheel

### DISCUSSION THREAD

**Discussion by lalo-mx** [IgniteUI/igniteui-angular#10942] [Q&A] [answered]
There is a new feature that allows inner elements to scroll during inertia wheel.
https://github.com/IgniteUI/igniteui-angular/pull/10793
Can we disable the new feature? There is bug https://github.com/IgniteUI/igniteui-angular/issues/10949 with a list (igxFor) with dynamic items(displayed only on mouse over) that does not allow to scroll down in the list even if there is not any scroll shown in the dynamic child.

**Comment by ChronosSF** [comment, marked as answer]
No way to disable it for now but we can adjust the implementation so that it doesn't break such cases. I'll provide you with more information when we have investigate it thoroughly.

**Comment by lalo-mx** [comment]
**Workaround - It could break easily with any new release:**
- Override didChildScroll to always return false.  https://github.com/IgniteUI/igniteui-angular/blob/19cd3adb8f151b5a77fb5184f6cb709d357f4e08/projects/igniteui-angular/src/lib/directives/scroll-inertia/scroll_inertia.directive.ts#L198
